windows/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -124,6 +124,9 @@
 
 - name: "Enable VBS"
   block:
+    - name: "Get guest OS system info"
+      include_tasks: ../utils/get_windows_system_info.yml
+      when: guest_os_system_info_retrieved is undefined or not guest_os_system_info_retrieved
```

Comment:
Looks like it is not necessary to add when here because this is a new deployed VM.

Reply:
Yes. Remove the condition. Thanks .